.github/workflows/docker_build.yml

```
@@ -1,65 +1,61 @@
 name: Build docker image and push to GHCR
 
 on:
-  schedule:
-    - cron: '0 0 * * *'   # Runs at 00:00 UTC every day
-  workflow_dispatch:  
+  push:
+  workflow_dispatch:
```

Todo: Change back before merging

---

```suggestion
  workflow_dispatch:
```

---

I think this is what we want, right?

```suggestion
  schedule:
    - cron: '0 0 * * *'   # Runs at 00:00 UTC every day
  workflow_dispatch:  
```

---

I worry that removal of the permissions block will prevent it from working on schedule. Right now, you as a maintainer do the pushes, so it runs using your token and permissions. But on schedule, we need to define the permission scope.

---

I've just moved the `permissions` block inside since the tutorial I followed put it there. I don't know what the difference is, should I move it back out?

I'm not a maintainer if you meant *me* in your comment